src/renderers/dom/fiber/ReactDOMFiber.js

```diff
@@ -197,6 +212,18 @@ var DOMRenderer = ReactFiberReconciler({
     return true;
   },
 
+  commitMount(
+    domElement : Instance,
+    type : string,
+    newProps : Props,
+    rootContainerInstance : Container,
+    internalInstanceHandle : Object,
+  ) : void {
+    if (shouldAutoFocusHostComponent(type, newProps)) {
+      (domElement : any).focus();
```

Comment:
You are no longer using the `focusNode` helper which means that this will start throwing in IE8 for hidden elements which is a breaking change. Do we really want that?